ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/DbChangelog/Update.hs

```diff
@@ -240,6 +240,10 @@ volatileStatesBimap f g =
 
 -- | Prune ledger states until at we have at most @k@ in the DbChangelog,
 -- excluding the one stored at the anchor.
+--
+-- >       0     | [ L0, L1, L2, L3, L4 ] | [ D1, D2, D3, D4 ]
+-- > >> prune (SecurityParam 3)
+-- >       0     | [         L2, L3, L4 ] | [ D1, D2, D3, D4 ]
```

Comment:
Might be good to specify/repeat what these columns are. Maybe just refer back to the documentation of `DbChangelog`?